packages/nvidia_nat_core/tests/nat/middleware/test_defense_middleware_content_guard.py

```
@@ -431,36 +432,25 @@ async def mock_next(_value):
-    async def test_target_location_validation(self, mock_builder, middleware_context):
-        """Test target_location validation and default behavior."""
-        # Test that target_location='input' raises ValidationError
-        from pydantic import ValidationError
-        with pytest.raises(ValidationError, match="Input should be 'output'"):
-            ContentSafetyGuardMiddlewareConfig(
-                llm_name="test_llm",
-                target_location="input",  # type: ignore[arg-type]
-                action="partial_compliance")
-
-        # Test default is 'output'
+    async def test_target_location_defaults_to_output(self, mock_builder, middleware_context):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_builder`

(ARG002)